client/js/nonprofits/donate/payment-step.js

```diff
@@ -170,10 +170,16 @@ const postTracking = (utmParams, donationResponse) => {
 }
 
 const postSuccess = (donationResponse) => {
-  // if (plausible) {
-  //   const resp = donationResponse()
-  //   plausible('payment_succeeded', {props: {amount: resp && resp.charge && resp.charge.amount && (resp.charge.amount / 100)}});
-  // }
+  try {
+    const plausible = window['plausible'];
+    if (plausible) {
+      const resp = donationResponse()
+      plausible('payment_succeeded', {props: {amount: resp && resp.charge && resp.charge.amount && (resp.charge.amount / 100)}});
+    }
+  }
+  catch(e) {
+    console.error(e)
+  }
```

Comment:
I wrap the whole plausible call in a try-catch. If something goes wrong, it won't block the UI from finishing properly.